peer_mentoring/account_management/templates/profile_index.html

```diff
@@ -10,10 +10,10 @@
         <div class="fw-bold"><a href="{% url 'account_management:view_profile' profile.user.id %}">{{ profile.user.username }}</a></div>
         <p class="card-text">{{profile.user.title}}</p>
       </div>
-      {% if profile.user != request.user %}
-      {% if profile.user not in request.user.userprofile.friends.all %}
+      {% if profile != request.user %}
+      {% if profile not in request.user.userprofile.friends.all %}
       <div>
-        <form method="post" id="friend_request" action="{% url 'account_management:send_request' user.id %}" style="display:inline;">
+        <form method="post" id="friend_request" action="{% url 'account_management:send_request' user_id=profile.user.id %}" style="display:inline;">
```

Comment:
nitpick (ignore for now) but you can use a bootstrap css class instead of `style="display:inline;"`